uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/crm/app.form.absenceperiod.js

```diff
@@ -44,6 +44,41 @@ define([
       this._validateOrder();
     };
 
+    /**
+     * Prompts the user via a modal to calculate entitlement for
+     * a new absence period
+     *
+     * @private
+     */
+    AbsencePeriod.prototype._showEntitlementCalculationPrompt = function () {
+      var confirmationMessage = 'The system will now update the staff members ' +
+                                'leave entitlement';
+
+      CRM.confirm({
+        title: ts('Update Leave Entitlement?'),
+        message: ts(confirmationMessage),
+        width: '30%',
```

Comment:
Is this an OK practice to store width in JS and not CSS?

Reply:
Well, not sure, I added this to keep it consistent with the other modal in the file. Should I remove it? 
https://github.com/compucorp/civihr/blob/cc485f95e976a4b1f1309b14d688b9d023cd1016/uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/crm/app.form.absenceperiod.js#L156